src/confluent_kafka/admin/__init__.py

```diff
@@ -109,16 +109,19 @@ class AdminClient (_AdminClientImpl):
     Requires broker version v0.11.0.0 or later.
     """
 
-    def __init__(self, conf):
+    def __init__(self, conf, logger=None):
```

Comment:
```suggestion
    def __init__(self, conf, **args):
```

Reply:
This will start taking all the properties in the conf as parameter. I don't know if we should support this.

Reply:
I was thinking of only supporting logger property like this.

Reply:
That's what happens with the Producer and Consumer too